src/sage/categories/rings.py

```diff
@@ -1646,6 +1647,40 @@ def _random_nonzero_element(self, *args, **kwds):
                 if not x.is_zero():
                     return x
 
+        def random_element(self, *args):
+            """
+            Return a random integer coerced into this ring.
+
+            INPUT:
+
+            - either no integer, one integer or two integers
+
+            The integer is chosen uniformly from the closed interval
+            ``[-2,2]``, ``[-a,a]`` or ``[a,b]`` according to the
+            length of the input.
+
+            ALGORITHM:
+
+            This uses Python's ``randint``.
+
+            EXAMPLES::
+
+                sage: ZZ.random_element(8)   # random
+                1
+                sage: QQ.random_element(8)   # random
+                2
+                sage: ZZ.random_element(4,12)   # random
+                7
+            """
```

Comment:
```suggestion
                sage: -8 <= ZZ.random_element(8) <= 8
                True
                sage: -8 <= QQ.random_element(8) <= 8
                True
                sage: 4 <= ZZ.random_element(4,12) <= 12
                True
            """
```
I think this is a better doctest.

I don’t know why this didn’t save before… :/